examples/pom.xml

```diff
@@ -167,6 +167,11 @@
                 <artifactId>shardingsphere-infra-common</artifactId>
                 <version>${shardingsphere.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.shardingsphere</groupId>
+                <artifactId>shardingsphere-read-write-splitting-api</artifactId>
```

Comment:
The `shardingsphere-read-write-splitting-api` has already change to `shardingsphere-readwrite-splitting-api`, please update it

Reply:
oh , oh , my fault .